cmd/controller-manager/app/controller_manager.go

```diff
@@ -142,6 +154,18 @@ func Run(controllerManagerOptions *options.ControllerManagerServer) error {
 	}
 	serviceCatalogKubeconfig.Insecure = controllerManagerOptions.ServiceCatalogInsecureSkipVerify
 
+	// Initialize SSL/TLS configuration.  Ensures we have a certificate and key to use.
+	// This is the same code as what is done in the API Server.  By default, Helm created
+	// cert and key for us, this just ensures the files are found and are readable and
+	// creates self signed versions if not.
+	secureServingOptions := genericserveroptions.NewSecureServingOptions()
+	secureServingOptions.ServerCert.CertDirectory = certDirectory
```

Comment:
`certDirectory` is not something that should be configurable?

Reply:
Yes, I agree, it should be.  Fixed.